scene/main/viewport.cpp

```diff
@@ -162,6 +162,24 @@ ViewportTexture::~ViewportTexture() {
 }
 
 void Viewport::_sub_window_update_order() {
+	if (gui.sub_windows.size() < 2) {
+		return;
+	}
+
+	if (!gui.sub_windows[gui.sub_windows.size() - 1].window->get_flag(Window::FLAG_ALWAYS_ON_TOP)) {
+		int index = gui.sub_windows.size() - 1;
+
+		while (index > 0 && gui.sub_windows[index - 1].window->get_flag(Window::FLAG_ALWAYS_ON_TOP)) {
+			--index;
+		}
+
+		if (index != (gui.sub_windows.size() - 1)) {
+			SubWindow sw = gui.sub_windows[index];
```

Comment:
The array index should be `gui.sub_windows.size() - 1`

Reply:
Sorry about that, still learning how git works exactly(I did git add before changing this line, then changed it and commited, my bad). Fixed.

Reply:
No worries. It takes a bit to get used to git.